crates/biome_module_graph/src/snapshots/biome_module_graph__module_graph__tests__resolve_export_types.snap

```diff
@@ -11,17 +11,93 @@ expression: data.exports
             local_name: Some(
                 "superComputer",
             ),
-            ty: Reference(
-                TypeReference {
-                    qualifier: TypeReferenceQualifier(
-                        [
-                            Borrowed(
-                                "DeepThought",
+            ty: Type(
```

Comment:
If you don't mind, I'd like to take a stab at these snapshots and make them a bit more readable. For example, it's very hard to understand what this snapshot belongs to (the source is missing). What do you think? 😄 Can I?